src/api.raml

```diff
@@ -510,7 +510,16 @@ traits:
                                         description: |
                                             Multiple versions including a version being previewed.
                                         value: !include samples/article-history/v1/unpublished-multiple-versions.json
-
+                                    preprintOnly:
```

Comment:
@nlisgo , do you think the inclusion of preprints should bump `article-history` from `v1` to `v2` ?

Reply:
if we bump this to `v2` there might be a bit more code/coordination required, but I could exclude preprint data from lax results so those requesting `v1` article-history data will continue working as normal and never receive preprints.

Reply:
I went ahead and bumped it. I think a rule of thumb when deciding to increment a spec version is "does it break things?" if yes, bump.